metamist/parser/generic_parser.py

```diff
@@ -190,6 +184,22 @@ def process_fieldname(self, fieldname: str) -> str:
         return fieldname.strip()
 
 
+def prepare_external_ids(
+    primary_external_id: str, external_ids: dict[str, str]
```

Comment:
If we need `or {}` below, then this probably wants to be `dict[str, str] | None`